packages/components/src/services/ForumService.ts

```diff
@@ -0,0 +1,55 @@
+import { HttpService } from '../utils/httpService';
+import { CreateQuestion, ForumPosts } from '../schemas/Forum.schema';
+
+const apiUrl = 'https://breakup-app-api.herokuapp.com/api';
+
+export const errorMessages = {
+  failedToLogin: 'Failed to login',
+} as const;
+
+export class ForumService {
+  constructor(private httpService: HttpService) {}
+
+  async fetchPosts(currentPage: number): Promise<ForumPosts[]> {
```

Comment:
Dont return promise, just return the data, change it through out the file

Reply:
ts warning deta hai, buyer app me bhi deta tha

Reply:
line number 36 par same kia hua h tune without promise